bindings/c/foundationdb/fdb_c.h

```diff
@@ -98,7 +98,7 @@ typedef struct key {
 	const uint8_t* key;
 	int key_length;
 } FDBKey;
-#if FDB_API_VERSION >= 710
+#if FDB_API_VERSION >= 720
```

Comment:
This number should not have changed from 630, so we may have had a couple releases that broke this.
```suggestion
#if FDB_API_VERSION >= 630
```